flow/envs/base.py

```diff
@@ -670,6 +677,18 @@ def terminate(self):
             # close pyglet renderer
             if self.sim_params.render in ['gray', 'dgray', 'rgb', 'drgb']:
                 self.renderer.close()
+            # generate video
+            elif (self.sim_params.render is True) and self.sim_params.save_render:
+                save_dir = os.path.expanduser('~') + '/flow_movies'
+                images_dir = self.path.split('/')[-1]
+                if not os.path.exists(save_dir):
+                    os.mkdir(save_dir)
+                speedup = 10  # speedup multiplier
+                fps = speedup//self.sim_step
```

Comment:
could you leave a comment explaining what this is? I can't quite figure it out from these lines alone.

Reply:
It adjusts the framerate so that you're watching `speedup` simulation seconds per 1 realtime second. I'll make the comment more explicit.